src/__swaps__/screens/Swap/components/GasButton.tsx

```diff
@@ -184,14 +181,11 @@ const GasMenu = ({ backToReview = false, children }: { backToReview?: boolean; c
         </ContextMenu>
       ) : (
         <ContextMenuButton
-          activeOpacity={0}
           enableContextMenu
-          isAnchoredToRight
           isMenuPrimaryAction
           menuConfig={menuConfig}
           onPressMenuItem={handlePressMenuItem}
           useActionSheetFallback={false}
-          wrapNativeComponent={false}
         >
```

Comment:
why were these props removed?

Reply:
added `isAnchoredToRight` back didn't notice it's used in the `.android`, the others are already the default inside `<ContextMenuButton />`

Reply:
btw seems like this GasButton menu is not working at all in android? idk if it's just my simulator